rs/execution_environment/src/canister_settings.rs

```diff
@@ -21,6 +21,7 @@ pub(crate) struct CanisterSettings {
     pub(crate) controllers: Option<Vec<PrincipalId>>,
     pub(crate) compute_allocation: Option<ComputeAllocation>,
     pub(crate) memory_allocation: Option<MemoryAllocation>,
+    pub(crate) wasm_memory_threshold: Option<NumBytes>,
```

Comment:
We now have two fields `wasm_memory_threshold` and `wasm_memory_limit`. Could you add some comments that clarify in what way they are different?

Reply:
Good point. Done.